src/python/nimbusml/pipeline.py

```diff
@@ -1816,22 +1819,43 @@ def _predict(self, X, y=None,
                 isinstance(X, DataFrame) and isinstance(y, (str, tuple))):
             y = y_temp
 
+        model_zip = ZipFile(self.model)
```

Comment:
are you unpacking zip file?

Reply:
its actually doesnt unpack the content, just reads the list of directories/entries,so it should not be intensive operation

---
In reply to: [314144315](https://github.com/microsoft/NimbusML/pull/230#discussion_r314144315) [](ancestors = 314144315)